solr/core/src/java/org/apache/solr/handler/component/HttpShardHandler.java

```diff
@@ -500,6 +505,28 @@ public void prepDistributed(ResponseBuilder rb) {
     }
   }
 
+  private static void forceDistributed(ResponseBuilder rb) {
+    SolrQueryRequest req = rb.req;
+    ModifiableSolrParams solrParams = new ModifiableSolrParams(req.getParams());
+    solrParams.set("shortCircuit", false);
+    req.setParams(solrParams);
+    if (req.getHttpSolrCall() != null
+        && StringUtils.isEmpty(req.getParams().get(ShardParams.SHARDS))) {
```

Comment:
It appears this would activate for SolrCloud... yet we wouldn't want that.
As I suggest below, by moving this logic to the one spot where we need it, then this whole portion of forceDistributed is removed.